Bottom: 259f66832be210fbc302722a05d63ac6536a605c
Top:    f5dd75d8422f3f1047ee8f650efda3f55407c00b
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-27 00:25:33 -0400

Refresh of mpc5200-ac97

---

diff --git a/sound/soc/fsl/mpc5200_psc_ac97.c b/sound/soc/fsl/mpc5200_psc_ac97.c
index bab80ac..4342b51 100644
--- a/sound/soc/fsl/mpc5200_psc_ac97.c
+++ b/sound/soc/fsl/mpc5200_psc_ac97.c
@@ -85,7 +85,7 @@ static void psc_ac97_warm_reset(struct snd_ac97 *ac97)
 	struct mpc52xx_psc __iomem *regs = psc_dma->psc_regs;
 
 	out_be32(&regs->sicr, psc_dma->sicr | MPC52xx_PSC_SICR_AWR);
-	spin_event_timeout(0, 3, 0);
+	udelay(3);
 	out_be32(&regs->sicr, psc_dma->sicr);
 }
 
@@ -95,9 +95,9 @@ static void psc_ac97_cold_reset(struct snd_ac97 *ac97)
 
 	/* Do a cold reset */
 	out_8(&regs->op1, MPC52xx_PSC_OP_RES);
-	spin_event_timeout(0, 10, 0);
+	udelay(10);
 	out_8(&regs->op0, MPC52xx_PSC_OP_RES);
-	spin_event_timeout(0, 50, 0);
+	udelat(50);
 	psc_ac97_warm_reset(ac97);
 }
